From 90d984c8ee72b4076edc40f3243ec215ac1baa8c Mon Sep 17 00:00:00 2001
From: Dinesh Karthikeyan <quic_dinek@quicinc.com>
Date: Tue, 16 May 2023 17:11:06 +0530
Subject: [PATCH] ath12k: Enable fwlog support for IPQ5332

Enable fwlog support for IPQ5332 by setting the hw_param en_fwlog to
true.

Signed-off-by: Dinesh Karthikeyan <quic_dinek@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/hw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath12k/hw.c b/drivers/net/wireless/ath/ath12k/hw.c
index 3b7039e..c039ee9 100644
--- a/drivers/net/wireless/ath/ath12k/hw.c
+++ b/drivers/net/wireless/ath/ath12k/hw.c
@@ -1700,7 +1700,7 @@ static const struct ath12k_hw_params ath12k_hw_params[] = {
 		.ce_remap = &ath12k_ce_remap_ipq5332,
 		.cmem_remap = &ath12k_cmem_ipq5332,
 		.send_platform_model = true,
-		.en_fwlog = false,
+		.en_fwlog = true,
 	},
 };
 
-- 
2.17.1

